src/main/java/org/opensearch/security/rest/SecurityHealthAction.java

```diff
@@ -64,9 +63,13 @@ public SecurityHealthAction(final Settings settings, final RestController contro
         this.registry = registry;
     }
 
-    @Override
     public List<Route> routes() {
-        return routes;
+        return ImmutableList.of();
```

Comment:
same

Reply:
My bad, they should be deleted as well, fixed